proxy-robots-module/src/test/java/orwell/proxy/ConfigFactoryTest.java

```diff
@@ -0,0 +1,68 @@
+package orwell.proxy;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.ConfigCli;
+import orwell.proxy.config.ConfigFactory;
+import orwell.proxy.config.EnumConfigFileType;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for {@link ConfigFactory}.
+ *
+ * @author miludmann@gmail.com (Michael Ludmann)
+ */
+
+@RunWith(JUnit4.class)
+public class ConfigFactoryTest {
+
+    final static Logger logback = LoggerFactory.getLogger(ConfigFactoryTest.class);
+    private static final String CONFIGURATION_FILE_TEST = "/configurationTest.xml";
```

Comment:
This is indeed a Java Resource, so the path is internally understood by the jvm.
Will keep it that way, changing FILE to RESOURCE in the name.